src/Packagist/Api/Result/AbstractResult.php

```diff
@@ -13,7 +13,26 @@ public function fromArray(array $data): void
         $inflector = InflectorFactory::create()->build();
         foreach ($data as $key => $value) {
             $property = $inflector->camelize($key);
+            if (null === $value && !$this->isNullable($property)) {
+                continue;
+            }
             $this->$property = $value;
         }
     }
+
+    private function isNullable(string $property): bool
+    {
+        if (PHP_MAJOR_VERSION < 8 || !property_exists($this, $property)) {
```

Comment:
Why do we need to check the PHP version here?

Reply:
Because the reflection feature is not available in PHP 7

Reply:
So maybe it's better to return `true` in this case, so for PHP 7 the behaviour will be the same as the old one.
About testing, it should be already covered in the current test suite, the only problem I see is that the actions workflow runs only on PHP 8.0 (while it should run on every supported PHP version)

Reply:
I agree that keeping the behaviour the same as it was for platforms that don't support the new changes is a good idea

Reply:
I've added PHP 7.4 to the CI builds here: https://github.com/KnpLabs/packagist-api/pull/90. You may need to rebase to pull in that change.

Reply:
I just changed the return value, squashed my commits and then rebased. I guess something has gone wrong since I see now two commits in this PR, but I'm confident that the superfluous commit will be ignored when merged